students/tony7kh/dom-practice/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DOM Practice</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="container">
+                <div class="container-wrap">
+                    <nav class="navbar">
+                        <a href="#" class="nav-branding">
+                            <img src="img/logo.png" alt="Audi">
+                        </a>
+                        <ul class="nav-menu">
+                            <li class="nav-item"><a href="#" class="nav-link" id="A4-button">A4</a></li>
+                            <li class="nav-item"><a href="#" class="nav-link" id="A5-button">A5</a></li>
+                            <li class="nav-item"><a href="#" class="nav-link" id="A8-button">A8</a></li>
+                            <li class="nav-item"><a href="#" class="nav-link" id="Q5-button">Q5</a></li>
+                            <li class="nav-item"><a href="#" class="nav-link" id="Q8-button">Q8</a></li>
+                        </ul>
+                        <div class="hamburger">
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                        </div>
+                    </nav>
+                </div>
+            </div>
+        </header>
+        <main class="main">      
+            <section class="section-banner">
```

Comment:
If you have just one common content inside `main` tag, you can skip `section` tag from semantic purpose.